orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/JenkinsTrigger.java

```diff
@@ -163,7 +168,7 @@ public int getNumber() {
     }
 
     public @Nonnull String getFullDisplayName() {
```

Comment:
Maybe we should just specify `@JsonIgnore` here

Reply:
Let's try that